src/components/Box/__stories__/Box.stories.mdx

```diff
@@ -0,0 +1,491 @@
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XS}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.XS</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.SMALL}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.SMALL</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.MEDIUM}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.MEDIUM</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.LARGE}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.LARGE</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XL}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.XL</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XXL}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.XXL</div>
+                </Box>
+            </div>
+            <div className={styles['monday-storybook-box-padding-wrapper']}>
+                <Box
+                    border={Box.borders.DEFAULT}
+                    padding={Box.paddings.XXXL}
+                >
+                    <div className={styles['monday-storybook-box-padding-inner']}>Box.paddings.XXXL</div>
+                </Box>
+            </div>
+        </div>
+    </Story>
+</Canvas>
+
+#### Property variations per each size:
+```
+padding
+paddingX (x axis: left, right)
+paddingY (y axis: top, bottom)
+paddingTop
+paddingEnd
+paddingBottom
+paddingStart
+```
+
+### Disabled
+<Canvas>
+    <Story name="Disabled">
+        <div className="monday-storybook-box-wrapper">
+            <Box
+                disabled>Box.disabled.DISABLED</Box>
+        </div>
+    </Story>
+</Canvas>
+
+## Related components
+
```

Comment:
let's add related components? Dialog container can be one, flex and responsive list are also options

Reply:
ok

Reply:
They are missing from the related component descriptions.
let sync on that in our meeting today.